var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -176,4 +176,10 @@ def flag_handler(self, name, flags):
                     flags = []
                 flags.append('-Wno-error=stringop-truncation')
 
+            if self.spec.satisfies('%gcc@8:') and \
+               self.spec.satisfies('@0.23.0:0.23.99'):
```

Comment:
Looks like this was fixed immediately after the 0.23.0 release.  So it doesn't apply to 0.23.1.

https://github.com/flux-framework/flux-core/commit/dd18b76fc79551d6368ec173926bfc43e4b59099

```suggestion
               self.spec.satisfies('@0.23.0'):
```

Reply:
Thanks!